code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -85,6 +85,12 @@
 	LAZYCLEARLIST(mobs_viewing)
 	return ..()
 
+/obj/item/storage/forceMove(atom/destination)
+	. = ..()
+	if(!ismob(destination.loc))
```

Comment:
In the event the `loc` *is* a mob, you want to make sure the UI is hidden for everyone **but** the mob itself.

Reply:
Fixed that [here](https://github.com/ParadiseSS13/Paradise/pull/17519/commits/318cb2c3744e9f2eb73f2dee6e9f13953039e829) (I think)